app/src/main/java/org/sopt/havit/MainApplication.kt

```diff
@@ -2,65 +2,28 @@ package org.sopt.havit
 
 import android.app.Application
 import com.kakao.sdk.common.KakaoSdk
-import org.koin.android.ext.koin.androidContext
-import org.koin.android.ext.koin.androidLogger
-import org.koin.android.viewmodel.dsl.viewModel
-import org.koin.core.context.startKoin
-import org.koin.dsl.module
-import org.sopt.havit.domain.repository.ContentsRepository
-import org.sopt.havit.domain.repository.MyPageRepository
-import org.sopt.havit.domain.repository.SearchRepository
-import org.sopt.havit.data.repository.ContentsRepositoryImpl
-import org.sopt.havit.data.repository.MyPageRepositoryImpl
-import org.sopt.havit.data.repository.SearchRepositoryImpl
-import org.sopt.havit.ui.mypage.MyPageViewModel
-import org.sopt.havit.ui.save.SaveViewModel
-import org.sopt.havit.ui.search.SearchViewModel
-import org.sopt.havit.ui.web.WebViewModel
+import dagger.hilt.android.HiltAndroidApp
 
-class MainApplication :Application() {
+@HiltAndroidApp
+class MainApplication : Application() {
 
-    companion object
-    {
+    companion object {
         var instance: MainApplication? = null
     }
```

Comment:
이건 그냥 질문..!! 이 `instance` 변수가 하는 일이 무엇인지 궁금해요!!

Reply:
안드로이드에서 코드를 짜다보면 context를 넘겨줘야 하는 부분이 많아서 불편한데 그때 여기 있는 intstance를 이용해 context를 해결한다고 하네요!!! 

```suggestion
    companion object {
        var instance: MainApplication? = null
        fun getMainApplicationContext(): Context {
            return instance!!.applicationContext
        }
    }
```

요렇게 바꿔놨어요~!!